proto/ddsketch/ddsketch.proto

```diff
@@ -0,0 +1,63 @@
+// Source: https://github.com/DataDog/sketches-go/blob/master/ddsketch/pb/ddsketch.proto
```

Comment:
Note: I had to copy this file from source & place it here because the [compiled version in sketches-go](https://github.com/DataDog/sketches-go/blob/master/ddsketch/pb/sketchpb/ddsketch.pb.go) is missing functions that the agent needs, so it was necessary for us to compile our own version of this file

Reply:
What functions are missing that agent needs?

I'm worried about this approach because it will create the need to conver between `github.com/DataDog/agent-payload/ddsketch.DDSketch` and  `sketchpb.DDSketch` on the backend.

Reply:
Can we update the compiled version in sketches-go?

Reply:
The agent needs `DDSketch.Size`, `DDSketch.MarshalTo` and `DDSketch.Unmarshal` (there might be more, but those are the functions that are initially causing the build to fail). And unfortunately, the difference between our version and the one in `sketches-go` is pretty substantial (you can see it [here](https://github.com/DataDog/sketches-go/compare/master...ISauve:patch-1)), so I'm not sure if our version would work for other projects which depend on `sketches-go`. If updating the source file is an option, I do think that would be best (in order to avoid the conversion in the back end that Lee mentioned)... I'll reach out to someone who manages the repo and ask what they think

Reply:
I confirmed with @piochelepiotr that re-generating `ddsketch.proto` in the sketches-go repo with `gogo proto` (that's the tool we use, they use `go proto`) would break other projects that are currently dependent on that file.

The only other solution I can think of would be to switch to using `go proto` to generate the protobufs in this repo - I believe that means that we'd have to update any files that use agent_logs_payload.pb.go, agent_payload.pb.go, and agent.pb.go though, which would end up being a significant agent & backend change.

Reply:
Changing the generate used here would almost certainly affect users of the agent-payload protos. 

Another idea is to use `repeated byte` as the type and have the backend do the protobuf parsing once there is parsed ConnectionMessage.